components/github_alert.lua

```diff
@@ -0,0 +1,89 @@
+local util = require 'util'
+local json = require 'json'
+local http_request = require 'http.request'
```

Comment:
Looks like it is not a built-in module in Lua, if so, please note what specific module is needed in README requirement section.